argocd/base/servicemonitors/argocd-repo-server

```diff
@@ -0,0 +1,13 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
This file should be a `.yaml` file.

Reply:
ok will fix and make a push, thanks!

Reply:
Yeah! Good catch! I just added it along with the prev fix